buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/measurement/TestLogFinder.groovy

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.gradle.plugins.measurement
+
+/** A utility class that finds test log links on different CI systems. */
+class TestLogFinder {
+
+    /** Returns the link to the current running test. */
+    static String getCurrentLogLink() {
```

Comment:
nit: `generateCurrentLogLink` sounds like a better fit since you are generating the url, but it's a matter of style.

Reply:
Done. Changed to `generateCurrentLogLink`.